test/validators.js

```diff
@@ -4698,12 +4698,54 @@ describe('Validators', () => {
       valid: ['abc', 'de', 'a', ''],
       invalid: ['abcd'],
     });
+    test({
+      validator: 'isLength',
+      args: [{ max: 6, exact: 5 }],
+      valid: ['abcde', 'fffde', 'allkk'],
+      invalid: ['abcd', 'abcdef', 'vfd', 'ff', '', 'k'],
+    });
+    test({
+      validator: 'isLength',
+      args: [{ max: 6, exact: { } }],
+      valid: [],
+      invalid: ['abcd', 'abcdef', 'vfd', 'ff', '', 'k'],
+    });
+    test({
+      validator: 'isLength',
+      args: [{ min: 2, max: 6, exact: 5 }],
+      valid: ['abcde', 'fffde', 'allkk'],
+      invalid: ['bsa', 'vfvd', 'ff', '', 'k'],
+    });
+    test({
+      validator: 'isLength',
+      args: [{ min: 1, exact: 2 }],
+      valid: ['fg', 'ff', 'po'],
+      invalid: ['bsa', 'vfvd', '', 'k'],
+    });
     test({
       validator: 'isLength',
       args: [{ max: 0 }],
       valid: [''],
       invalid: ['a', 'ab'],
     });
+    test({
+      validator: 'isLength',
+      args: [{ min: 5, max: 10, exact: [2, 6, 8, 9] }],
+      valid: ['helloguy', 'shopping', 'validator', 'length'],
+      invalid: ['abcde', 'abcdefg'],
+    });
+    test({
+      validator: 'isLength',
+      args: [{ min: 5, max: 10, exact: { first: 6, second: 8, third: 9 } }],
+      valid: ['helloguy', 'shopping', 'validator', 'length'],
+      invalid: ['abcde', 'abcdefg', 'abcdefghij'],
+    });
+    test({
+      validator: 'isLength',
+      args: [{ exact: '9' }],
+      valid: [],
+      invalid: ['a', 'abcd', 'abcdefghijkl'],
+    });
```

Comment:
Regarding my remarks above, this would have to change:

You should have a test to cover for the case when no `args` are provided.

```suggestion
test({
      validator: 'isLength',
      args: [],
      valid: ['a', 'abcd', 'abcdefghijkl'],
      invalid: [],
    });
```

You also need a test to assert the situation when only the `exact` arg is provided, but is not an Array:

```suggestion
    test({
      validator: 'isLength',
      args: [{ exact: '9' }],
      valid: ['a', 'abcd', 'abcdefghijkl'],
      invalid: [],
    });
```